app/helpers/copies_helper.rb

```diff
@@ -1,2 +1,5 @@
 module CopiesHelper
```

Comment:
Having this method in a helper is good but it doesn't really have anything to do with Copies. You can imagine if we expanded the application we might want to make admin only endpoints for Books and Users and other classes too. Something like `PermissionsHelper` sounds better to me.